source/connect.txt

```diff
@@ -38,133 +37,118 @@ This is equivalent to the following command:
-     your connection string from the |service| UI. To learn more, see 
-     :atlas:`Connect to a Cluster </connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`.
+.. code-block:: sh
 
-- The command-line options ``--host`` and ``--port``. If you do not 
-  include the ``--port`` option, ``mongosh`` uses the **default port** 
-  27017.
+   mongosh "mongodb://mongodb0.example.com:28015"
 
-  .. example:: 
+.. code-block:: sh
 
-     To connect to a MongoDB
-     instance running on a remote host on port 28015:
+   mongosh --host mongodb0.example.com --port 28015
 
-     .. code-block:: sh
+.. note:: Connect to |service-fullname|
 
-        mongosh --host mongodb0.example.com --port 28015
+   If your remote host is a |service| cluster, you can copy your
+   connection string from the |service| UI. To learn more, see
+   :atlas:`Connect to a Cluster
+   </connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`
+   in the Atlas documentation.
 
 Connection Options
 ------------------
 
-Connect with Authentication
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To connect to a MongoDB instance requires authentication, use the 
-``--username`` and ``--authenticationDatabase`` command-line options. 
-``mongosh`` prompts you for a password, which it masks as you type.
-  
-.. example:: 
+Specify different connection options to connect to different types of
+deployments.
 
-   To connect to a remote MongoDB instance and authenticate against 
-   the ``admin`` database as user ``alice``:
+Connect With Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. code-block:: sh
+To connect to a MongoDB deployment requires authentication, use the
+:option:`--username <--username>` and :option:`--authenticationDatabase
+<--authenticationDatabase>` options. ``mongosh`` prompts you for a
+password, which it hides as you type.
 
-      mongosh "mongodb://mongodb0.example.com:28015" --username alice --authenticationDatabase admin
+For example, run the following command to authenticate on the ``admin``
+database as user ``alice``:
 
-.. note::
+.. code-block:: sh
 
-   To provide a password with the command instead of using the masked 
-   prompt, you can use the ``--password`` option.
+   mongosh "mongodb://mongodb0.example.com:28015" --username alice --authenticationDatabase admin
 
 .. seealso::
 
-   - :manual:`Enable Access Control </tutorial/enable-authentication/>` 
-     to enforce authentication.
-   - Add :manual:`Database Users </core/security-users/>` 
-     to provide authenticated access to a MongoDB instance.
+   - To enforce authentication on a deployment, see
+     :manual:`Enable Access Control </tutorial/enable-authentication/>`.
+
+   - To provision access to a MongoDB deployment, see :manual:`Database
+     Users </core/security-users/>`.
 
 Connect to a Replica Set
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-To connect to a replica set:
+To connect to a replica set, perform one of the following actions:
```

Comment:
[suggestion] I'm finding this structure a little confusing. What are the two options here? Should each option have its own heading?